src/components/Account/Login.vue

```diff
@@ -139,7 +139,8 @@ export default {
 
       try {
         await loginFunc()
-        this.$router.back()
+        const redirect = this.getRedirectPath
+        if (redirect) { this.$router.push({ path: redirect.redirect }) } else { this.$router.back() }
```

Comment:
If the router starts at '/login', then there's no entry in the history stack. `this.$router.back()` won't work.
Maybe add more checks to prevent this behavior? (e.g. by checking [`router.START_LOCATION`](https://v3.router.vuejs.org/api/#router-start-location-3-5-0))